pkg/operator/vttablet/pod.go

```diff
@@ -338,6 +333,18 @@ func UpdatePod(obj *corev1.Pod, spec *Spec) {
 	obj.Spec.PriorityClassName = vttabletPriorityClassName
 }
 
+func updatePodAnnotations(obj *corev1.Pod, spec *Spec) {
```

Comment:
This is fine, but for some reason I sorta like the idea of forgoing DRY here and distinctly expressing everything inside the strategy handler functions. Do we ever foresee annotations diverging between strategies after this?